<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QProfileView</class>
 <widget class="QMainWindow" name="QProfileView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>490</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QProfileView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Img">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StopCapture">
    <property name="geometry">
     <rect>
      <x>222</x>
      <y>405</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>重置区域</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CaptureDone">
    <property name="geometry">
     <rect>
      <x>374</x>
      <y>405</y>
      <width>421</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>完成设置</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Count">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>405</y>
      <width>61</width>
      <height>35</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StartCapture">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>405</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
